contents.json

```diff
@@ -17870,6 +17870,28 @@
       "date_added": "Jul 5 2019",
       "suggested_by": "@dotwee",
       "stars": 65
+    },
+    {
+      "title": "Unstoppable Wallet",
+      "description": "A fully decentralized Bitcoin, Ethereum, EOS, Binance Chain, Bitcoin Cash and DASH currency wallet",
+      "category-ids": [
+        "cryptocurrency"
+      ],
+      "source": "https://github.com/horizontalsystems/unstoppable-wallet-ios",
+      "itunes": "https://itunes.apple.com/app/bank-bitcoin-wallet/id1447619907?ls=1&mt=8",
+      "license": "mit",
+      "stars": 89,
+      "tags": [
+        "swift",
+      ],
+      "lang": "eng",
```

Comment:
I don't think this is necessary, can you remove this line with `lang` please

Reply:
done!